logger.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"golang.zx2c4.com/wireguard/device"
+)
+
+// Use device package logger as the global logger for the application.
+// device.Logger is effectively a collection of standard logging library loggers
+var logger *device.Logger
+
+// logLevel global var, defaults to info
+var logLevel = device.LogLevelInfo
+
+// Sets the logLevel global variable. Needs to be called before the
+// initialisation of loggers
+func setLogLevel(level string) {
+	switch level {
+	case "debug":
+		logLevel = device.LogLevelDebug
+	case "info":
+		logLevel = device.LogLevelInfo
+	case "error":
+		logLevel = device.LogLevelError
+	}
+}
+
+// Returns a new logger using the global level variable
+func initLogger(name string) *device.Logger {
```

Comment:
Since this is returning a new `device.Logger` maybe we should call it `newLogger` instead?